test/forge/MetaMorphoInternalTest.sol

```diff
@@ -31,6 +31,7 @@ contract MetaMorphoInternalTest is InternalTest {
 
         Id id = allMarkets[0].id();
         _setCap(id, CAP);
+        supplyQueue = [id];
```

Comment:
why do we need this?

Reply:
because here we are in metamorpho itself so to replicate the previous behavior we do this